client/js/_player/components/common/clix_drag_and_drop/clix_drag_and_drop.jsx

```diff
@@ -0,0 +1,220 @@
+import React  from 'react';
+import _      from 'lodash';
+
+import withDragDropContext  from '../with_drag_drop_context';
+import ClixDropZone         from './clix_drop_zone';
+import Droppable            from './droppable'; // eslint-disable-line import/no-named-as-default
+import CustomDragLayer      from '../custom_drag_layer'; // eslint-disable-line import/no-named-as-default
+
+export class ClixDragAndDrop extends React.Component {
+  static propTypes = {
+    answers: React.PropTypes.arrayOf(React.PropTypes.shape({
+      id: React.PropTypes.string, // eslint-disable-line react/no-unused-prop-types
```

Comment:
im going to vote that in this scenario we just remove the propTypse that 'arent used' for the sake of code clarity. Looking through the rest of the code it looks like the child components have sufficient prop validation that we will know if there is an issue with one of these props.